### ISSUE TITLE
Pull Requests queue

### ISSUE THREAD

**Issue by randolf** [trusteddomainproject/OpenDMARC#253]
Does anyone know how to contact someone to look into the Pull Requests queue?  There are a lot of contributions that would be great to see implemented in this project.  Thanks.

**Comment by dgeo** [comment]
I'd propose to merge @andreasschulze's https://github.com/andreasschulze/OpenDMARC/tree/builder branch for a start

**Comment by andreasschulze** [comment]
no, don't do that. My branch `builder` is indented to be a "final result" for private purposes.

I've to better document, which changed against `trusteddomainproject/opendmarc:develop` I applied.
Then *this* changes should be merged to `trusteddomainproject/opendmarc:develop` separately

**Comment by thegushi** [comment]
I am attempting to go through the backlog, as well as give the project some other love as well.  Andreas, your ideas are welcome.  If you want a secondary contact method, please let me know.

Randolf, I added some comments on your postgres issue as well.
